modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -218,51 +479,73 @@ cv::Vec3i HashTSDFVolumeCPU::volumeToVoxelCoord(cv::Point3f point) const
                      cvFloor(point.z * voxelSizeInv));
 }
 
+inline TsdfVoxel HashTSDFVolumeCPU::_at(const cv::Vec3i& volumeIdx, volumeIndex indx, 
+    Point3i volResolution, Vec4i volStrides1) const
```

Comment:
Extra argument `volStrides1`, `this->volStrides` is the same everywhere and is never changed

Reply:
The same about volume resolution
Since the check is removed, `volResolution` is not needed too

Reply:
done

Reply:
done